pkg/signup/signup_service.go

```diff
@@ -154,6 +173,13 @@ func (s *ServiceImpl) CreateUserSignup(ctx *gin.Context) (*crtapi.UserSignup, er
 	return created, nil
 }
 
+func splitEmail(email string) (account, host string) {
```

Comment:
do we need to return the account? seems like it's ignored when we call `splitEmail`

Reply:
I refactored this slightly.